src/plugins/discover/public/application/main/components/top_nav/open_alerts_popover.tsx

```diff
@@ -54,6 +60,13 @@ export function AlertsPopover({
    * Provides the default parameters used to initialize the new rule
    */
   const getParams = useCallback(() => {
+    if (isPlainRecord) {
+      return {
+        searchType: 'esqlQuery',
+        esqlQuery: query,
+        timeField,
```

Comment:
Under the textarea it says that `@timestamp is not detetected` for some reason:

<img width="600" alt="Screenshot 2023-09-12 at 11 12 31" src="https://github.com/elastic/kibana/assets/1415710/962f9e2a-f6a7-4ba0-a457-5ad8cb8cdc05">

Reply:
The problem is again on the stack alerts plugin but I will try to solve this one

Reply:
@jughosta fixed that https://github.com/elastic/kibana/pull/165973/commits/646439785c62e12157e1d83627a19500ee44be3c